src/routes/private.route.tsx

```diff
@@ -1,11 +1,17 @@
 import { composeModuleRoutes } from '#/shared/utils/router.util';
 import { WishlistRoutes } from '#/modules/wishlist';
 import { ProtectedLayout } from '#/layout/components/protected-layout';
+import { UserAccommodationsRoutes } from '#/modules/owner';
 
 export const privateRoutes = [
   {
     path: '/',
     element: <ProtectedLayout />,
     children: composeModuleRoutes(WishlistRoutes),
   },
+  {
+    path: '/',
+    element: <ProtectedLayout />,
+    children: composeModuleRoutes(UserAccommodationsRoutes),
```

Comment:
I think that we can move UserAccommodationsRoutes in the upper element:

```tsx
{
    path: '/',
    element: <ProtectedLayout />,
    children: composeModuleRoutes(WishlistRoutes, UserAccommodationsRoutes),
  },
  ```
  
  Also maybe rename the router name. E.g. OwnerRoutes